integrations/langfuse/tests/test_tracing.py

```diff
@@ -19,6 +19,36 @@
 os.environ["HAYSTACK_CONTENT_TRACING_ENABLED"] = "true"
 
 
+@pytest.fixture
+def pipeline_with_env_vars(llm_class, expected_trace):
+    """Pipeline factory using environment variables for Langfuse authentication"""
+    pipe = Pipeline()
+    pipe.add_component("tracer", LangfuseConnector(name=f"Chat example - {expected_trace}", public=True))
+    pipe.add_component("prompt_builder", ChatPromptBuilder())
+    pipe.add_component("llm", llm_class())
+    pipe.connect("prompt_builder.prompt", "llm.messages")
+    return pipe
+
+
+@pytest.fixture
+def pipeline_with_secrets(llm_class, expected_trace):
+    """Pipeline factory using Secret objects for Langfuse authentication"""
```

Comment:
Same as above, let's use monkeypatch.setenv

Reply:
Nevermind. Thought it's a unit test but it's a fixture used in an integration test.